community/slack.md

```diff
@@ -1 +1,69 @@
+* **#feed-software-review:** this channel tracks new review submissions in our software-submission GitHub repo. This feed helps pyOpenSci staff and its editorial team keep track of the cadence of submissions in a centralized location.
+* **#github-issues:** this channel was created to make it easier to follow issues and pull requests opened in our core Github repositories, as given the volume of activity it’s easy to miss new items.
+
+## Setting up a Slack bot using Slack Workflows
+Administrators in the pyOpenSci Slack space can set up [workflows](https://slack.com/help/articles/16583775096083-Automations--What-is-a-Slack-workflow), which are automated bots that complete tasks based on automatic triggers. We currently use a workflow to onboard new members to the Slack space, and if there are additional bots you feel would be helpful, please reach out to Leah or Jesse.
+
+## Setting up an automatic feed to post from GitHub into Slack:
+* First, grant permission to the Slack app from within GitHub. Then you can customize which Slack channels you want to use in order to track new activity on GitHub.
+* To use this integration, you can provide commands directly in the channel to connect it to a specific feed of events on GitHub. Some useful commands are:
+    * **`/GitHub subscribe pyopensci/python-package-guide issues`** will allow you to subscribe to issues in the python-package-guide channel
+    * **`/GitHub unsubscribe pyopensci/python-package-guide pulls`** will unsubscribe you
+    * **`/GitHub help`** provides you with all of the options in terms of subscribing and unsubscribing
+
+## Slack’s limitations - nonprofit plan
+pyOpenSci currently has a Slack paid plan that is provided at no cost for nonprofits. It allows up to 250 users and unlimited saved messages. While this solution currently meets our needs, it will become a challenge in the future as pyOpenSci continues to grow. The main concern is the loss of message history.
+
+We initially chose Slack as most of our community members already had organizations on Slack, and many did not like Discord. As we look ahead and consider our future needs, it’s worth noting that some of our partners have moved to platforms such as Mattermost and Zulip.
+
+## Slack for pyOpenSci Community Discussion
+Slack is also used for broader pyOpenSci community discussion. However, it is currently invitation-only, which models the same processes used by rOpenSci. An invite-only model is important as pyOpenSci grows because:
+* It allows us to grow carefully and slowly
+* It allows us to create an inclusive environment that is evolving into a self-moderating community. This is significantly more challenging to do in an online community with large influxes of members.
+* Everyone invited to Slack has generally already contributed to pyOpenSci in some way, and therefore have a vested interest in the community upon joining.
+### Who we invite to the community:
```

Comment:
```suggestion
* Everyone invited to Slack has generally already contributed to pyOpenSci in some way, and therefore have a vested interest in the community upon joining.

### Who we invite to the community:
```

Reply:
it's a good idea to have a full line break after a section heading. can you please fix this throughout? sphinx might allow it but generally we want a return before a new heading for readability too!

Reply:
yes! I'll get that sorted - thanks for flagging!